lib/Distribution/Opensuse/Tumbleweed.pm

```diff
@@ -213,4 +214,8 @@ sub get_dns_server_setup {
     return YaST::DNSServer::DNSServerSetupController->new();
 }
 
```

Comment:
please add a comment:
`This section below only use Pages, and avoid the use of unnecessary Controllers`